### PR TITLE
[6.14.z] Add test to ensure katello-agent availability in Client repos

### PR DESCRIPTION
**Pull request by vsedmik**
Katello-agent was deprecated and removed from 6.15 and upstream repos, but we still need to have it available in Client repos for older Satellite versions which support it. This test just verifies that.

### CODE REVIEW

**Comment by Griffin-Sullivan** [comment]
trigger: test-robottelo
pytest: tests/foreman/api/test_repository.py -k test_positive_katello_agent_availability

**Review by pondrejk — APPROVED** [review]

**Review by synkd — APPROVED** [review]

File: robottelo/constants/__init__.py [hunk 1]
```diff
@@ -289,9 +289,9 @@ class Colored(Box):
     'rhsc7': 'Red Hat Satellite Capsule 6.11 (for RHEL 7 Server) (RPMs)',
     'rhsc8': 'Red Hat Satellite Capsule 6.13 for RHEL 8 x86_64 (RPMs)',
     'rhsc7_iso': 'Red Hat Satellite Capsule 6.4 (for RHEL 7 Server) (ISOs)',
-    'rhsclient7': 'Red Hat Satellite Client 6 for RHEL 7 Server RPMs x86_64',
-    'rhsclient8': 'Red Hat Satellite Client 6 for RHEL 8 x86_64 RPMs',
-    'rhsclient9': 'Red Hat Satellite Client 6 for RHEL 9 x86_64 RPMs',
+    'rhsclient7': 'Red Hat Satellite Client 6 (for RHEL 7 Server) (RPMs)',
+    'rhsclient8': 'Red Hat Satellite Client 6 for RHEL 8 x86_64 (RPMs)',
+    'rhsclient9': 'Red Hat Satellite Client 6 for RHEL 9 x86_64 (RPMs)',
```
Comment (Gauravtalreja1) [inline robottelo/constants/__init__.py:294]: @vsedmik we're missing this updates in master too, not sure of older .z branches, so can you update it in separate it PR please to auto-cherrypick?

File: tests/foreman/api/test_repository.py [hunk 2]
```diff
@@ -1295,6 +1295,42 @@ def test_positive_sync_with_treeinfo_ignore(
...
+        :id: cd5b4e5b-e4f1-4d00-b171-30cd5b1e7ce8
+
+        :parametrized: yes
+
+        :steps:
+            1. Enable RH Satellite Client repo and sync it.
+            2. Read the repo packages.
+
+        :expectedresults:
+            1. Katello-agent remains available in the Client repos.
```
Comment (Gauravtalreja1) [inline tests/foreman/api/test_repository.py:1315]: Also, I know client repo is common to all  .z branches, but shouldn't we verify if this exists there too? can you update labels accordingly

Comment (vsedmik) [reply tests/foreman/api/test_repository.py:1315]: Well, my idea was to test it just until `6.14.z` (the last version using katello-agent) is supported/tested. I expect `6.14.z` releases be more frequent than the former z-releases, so I think we are good with 6.14.z.

File: robottelo/constants/__init__.py (hunk 1 above)
Comment (vsedmik) [reply robottelo/constants/__init__.py:294]: Sure, I can raise a PR to update it in `master`.

**Comment by vsedmik** [comment]
Split into two commits as per @Gauravtalreja1's request.

**Review by Gauravtalreja1 — APPROVED** [review]
Ack, thank you @vsedmik :)

**Pull request merged**